src/components/gallery/gallery.scss

```diff
@@ -54,8 +54,13 @@
       align-self: center;
       width: 33.3%;
 
-      &::before {
-        font-size: 20px;
+      //&::before {
```

Comment:
Wasn't going to say anything but then I had the other comment about the button label anyways...So if you're updating...please delete unused/commented out code